File: src/UpSchool.Domain/Dtos/GeneratePasswordDto.cs
namespace UpSchool.Domain.Dtos
{
    public class GeneratePasswordDto
    {
        public bool IncludeNumbers { get; set; } = true;
        public bool IncludeLowercaseCharacters { get; set; } = true;
        public bool IncludeUppercaseCharacters { get; set; }
        public bool IncludeSpecialCharacters { get; set; }
        public int Length { get; set; } = 6;
    }
}

File: react-upstorage/src/types/GeneratePasswordDto.ts

export class GeneratePasswordDto{
    public IncludeNumbers:boolean;
    public IncludeLowercaseCharacters:boolean;
    public IncludeUppercaseCharacters:boolean;
    public IncludeSpecialCharacters:boolean;
    public Length:number;

    constructor() {
        this.IncludeNumbers = true;
        this.IncludeLowercaseCharacters = true;
        this.Length = 6;
        this.IncludeUppercaseCharacters=false;
        this.IncludeSpecialCharacters = false;
    }
}